ledger-core/v2/conveyor/smachine/slot.go

```diff
@@ -531,22 +531,22 @@ func (s *Slot) logStepError(action ErrorHandlerAction, stateUpdate StateUpdate,
 	if area.IsDetached() {
 		flags |= StepLoggerDetached
 	}
-	s._logStepUpdate(StepLoggerUpdate, 0, durationUnknownNano, durationUnknownNano, stateUpdate, flags, err)
+	s._logStepUpdate(StepLoggerUpdate, durationUnknownNano, durationUnknownNano, stateUpdate, flags, err)
```

Comment:
Just out of curiosity - does underscore has any special meaning?

Reply:
yes, I use it to denote a class-private method equivalent, e.g. a method that can only be called inside a mutex. This helps me to avoid accidental mistakes.